src/System.ServiceModel.Duplex/src/System.ServiceModel.Duplex.csproj

```diff
@@ -19,10 +19,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
 
   <ItemGroup>
-    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' != 'netcore50aot'" />
-    <TargetingPackReference Include="System.ServiceModel" Condition="'$(TargetGroup)' == 'net46'" />
-    <ProjectReference Include="$(SourceDir)System.Private.ServiceModel\src\System.Private.ServiceModel.csproj" Condition="'$(TargetGroup)' != 'net46'" >
-      <AdditionalProperties>OSGroup=Windows_NT</AdditionalProperties>
+    <ProjectReference Include="$(SourceDir)System.Private.ServiceModel\src\System.Private.ServiceModel.csproj" >
```

Comment:
Hmm, does it work differently on my own box than wherever we build the official packages? I don't see S.P.SM getting built twice on my machine, just once for Windows.